src/main/java/net/openhft/chronicle/wire/VanillaMethodWriterBuilder.java

```diff
@@ -118,6 +118,7 @@ public MethodWriterBuilder<T> addInterface(Class additionalClass) {
         return this;
     }
 
+    @Deprecated(/* Replaced by UpdateInterceptor. To be removed in x.24 */)
```

Comment:
I know we normally give two releases warning, but this has not been used internally since https://github.com/OpenHFT/Chronicle-Wire/issues/159. If any customers are using this, it is straightforward to replace with UpdateInterceptor